docs/en/reference/transactions.rst

```diff
@@ -28,11 +28,11 @@ is functionally equivalent to the previous one:
 ::
 
     <?php
-    $conn->transactional(function($conn) {
+    $conn->transactional(function(Connection $conn): void {
```

Comment:
The closure does not have to be a `void`, right? Isn't the returned value passed through by `transactional()`? Should we mention that?

Reply:
Oh right sorry, I misread the code

Reply:
I guess since there's no return statement in this closure and the return value isn't used anywhere, it's void. Maybe it makes sense to have two examples: one with a return value and one without (both are valid).

Reply:
I added a note paraphrasing what @derrabus said, not sure if you saw it before writing that comment and if you think two examples are better, in which case I'll be happy to oblige :slightly_smiling_face:

Reply:
I started writing my comment before he had posted his but I submitted it after I saw your update. I still believe that two examples are better than just an explanation, they complement each other, not exclude.

Reply:
@derrabus Does this resolve your question/suggestion?